Finished project

Doc415 opened the pull request.

Doc415 committed:
Finished project

Doc415 committed:
Update Home.razor

changed oninitialized to oninitializedasync to wait records from db

TheCSharpAcademy reviewed (changes requested):
@Doc415 Is the project running on for you? I'm getting this upon starting it:
![image](https://github.com/TheCSharpAcademy/CodeReviews.Blazor.Wardrobe/assets/108711476/75653866-7f4a-496b-8db8-0ff822175a93)

Doc415 commented:
> @Doc415 Is the project running on for you? I'm getting this upon starting it: ![image](https://private-user-images.githubusercontent.com/108711476/312275198-75653866-7f4a-496b-8db8-0ff822175a93.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyOTg3MzQsIm5iZiI6MTcxMDI5ODQzNCwicGF0aCI6Ii8xMDg3MTE0NzYvMzEyMjc1MTk4LTc1NjUzODY2LTdmNGEtNDk2Yi04ZGI4LTBmZjgyMjE3NWE5My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxM1QwMjUzNTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mN2I3ZWNmMDM2Mzc5ODQ0NDIzNWIwNWUyYWQ3NmFmZDE0OTdiMmE0YmY3OTFiN2NjNGRhZjE5YmQ4ODBlMTY3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.4ML1uN2pRyH07O9SxLD0fbBeBjQbjgVFQz7fggUUqUw)

It was working at the time I sent PR but now i am also not able to run it. 
Changed the JS scripts to CDN but it seems like there is a problem in the sqlitewasm library
I ll check it

Doc415 commented:
I have fixed a using and i updated VS and it is working on my side.
You need  webassembly tools to be installed to run the app
![image](https://github.com/TheCSharpAcademy/CodeReviews.Blazor.Wardrobe/assets/94632226/80434291-7165-4eeb-ba9e-27bed9d976b7)
![image](https://github.com/TheCSharpAcademy/CodeReviews.Blazor.Wardrobe/assets/94632226/9cf2ccb2-a668-4188-88dd-c695bf96b8e3)

Doc415 committed:
Fixed using

Doc415 committed:
minor card updates
